bazel/python_rules.bzl

```diff
@@ -153,7 +183,12 @@ _generate_pb2_grpc_src = rule(
             providers = [ProtoInfo],
         ),
         "strip_prefixes": attr.string_list(),
-        "_plugin": attr.label(
+        "plugin": attr.label(
+            mandatory = False,
+            executable = True,
+            cfg = "host",
```

Comment:
Any reason why no specific provider?

Reply:
There was no particular reason. Added `providers = ["files_to_run"],` to stay consistent with the other attributes like this.